app/templates/closed.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block title %}Closed - My Study{% endblock title %}
+
+{% block content %}
+
+<h1 class="saturn" data-ga="info" data-ga-category="info" data-ga-action="info-message-shown" data-ga-label="collection-exercise-closed">
```

Comment:
@jcox-dev The PL analytics script only looks for data-ga  values of 'click', 'error' and 'visible', so ga won't send any events here with this 'info' value.  
'Visible' keeps checking and sending events every 2 seconds, so we'll have to use 'error' here.  The action and category can stay the same though, so it'll still be easy to read in ga.

Reply:
Great. Cheers, @jrbarnes9.